### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by ivababukova**
# Description
We noticed that the readme for the python part of the worker is really out of date.

# Details
#### URL to issue
N/A

#### Link to staging deployment URL (or set N/A)
N/A

#### Links to any PRs or resources related to this PR
N/A

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [ ] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [ ] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [ ] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [ ] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [ ] Started end-to-end tests on the latest commit.

### Documentation updates
- [ ] Relevant Github READMEs updated **or** no GitHub README updates required.
- [ ] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [ ] Staging environment is unstaged before merging.
- [x] Photo of a cute animal attached to this PR.
duck

### CODE REVIEW

**Comment by ivababukova** [comment]
duck

File: README.md [hunk 1]
```diff
@@ -102,7 +64,7 @@ extension installed. Make sure you restart VS Code after installing to make sure
 loads successfully. You should see a green icon in the leftmost part of the status bar,
 which indicates that the remote container plugin has been installed.
 
-#### Git LFS
+### Git LFS
 File(s) under `data/test` are downloaded by [inframock](https://github.com/hms-dbmi-cellenics/inframock), uploaded to mock S3 and used by the workers. As some of these files are over Github's file size limit (100 MB), they are stored using [Git LFS](https://git-lfs.github.com/). Follow the installation instructions on their website to setup Git LFS locally.
 
 Once you have installed Git LFS, you can open the worker root directory in a terminal and run
```
Comment (aerlaut) [inline README.md]: We don't use Git LFS anymore, so we can remove this

File: README.md [hunk 2]
```diff
@@ -137,10 +99,53 @@ These development environments should be pre-configured with the same requiremen
 produciton instances, as well as the necessary VS Code extensions required to debug and
 lint code.
 
+## Running locally
```
Comment (aerlaut) [inline README.md:87]: Can we start with `Step 0. Prerequisites` like we have in the API? We'll need to have inframock up and running to do be able to run workers too right?

Comment (ivababukova) [reply README.md:87]: not necessary, you can debug worker-only code without having it connected to the rest of the platform. You can submit custom work to it via another terminal (see the python readme, `pushing custom work to the local worker` section).

Comment (ivababukova) [reply README.md:87]: I removed the prerequisites step, because it was empty

File: README.md (hunk 1 above)
Comment (ivababukova) [reply README.md]: good catch, thanks!

File: r/README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 worker-r
 ========
 
-The single-cell pipeline work executor for R.
+The Cellenics single cell data analysis tasks executor, written in R.
```
Comment (alexvpickering) [inline r/README.md:5]:
```suggestion

Overview
--------
The R part of the worker runs the data analysis tasks. It fullfills the following functions:
- Loads the processed `SeuratObject` for the experiment id assigned to the pod, reloading it if the associated file changes.
- Creates a local REST API specific to the assigned experiment id with [RestRserve](https://restrserve.org/index.html) that listens for requests from the Python worker.
- Runs the task function specified in the request from the Python worker.
- Returns the results back to the Python worker.
```

File: README.md [hunk 4]
```diff
@@ -2,17 +2,27 @@
...
 
-## Running locally
-To run the worker locally, you will need to build it and then run it, passing
-the id of the processed experiment that you want to use the worker with.
+More specific details about the Python or the R part of the worker can be found in the README file in the respective folder (python/ or r/).
+
+## Deployment
 
-### 0. Prerequisites
+The API is deployed as a Helm chart to an AWS-managed Kubernetes cluster and runs on a Fargate-managed node. The Helm chart template for the worker is located in `chart-infra/` folder.
```
Comment (aerlaut) [inline README.md]:
```suggestion
The worker is deployed as a Helm chart to an AWS-managed Kubernetes cluster and runs on a Fargate-managed node. The Helm chart template for the worker is located in `chart-infra/` folder.
```

File: README.md [hunk 5]
```diff
@@ -2,17 +2,27 @@
 worker
 ======
 
-The single-cell pipeline work executor. It consists of two containers: a Python container and an R container. 
+The Cellenics data analysis tasks executor. It consists of two containers: a Python container and an R container. For more details on the individual containers, check out the README files in their respective directories.
 
-The Python part of the worker is a wrapper around the R part: it receives tasks from the API, parses them, sends them to the R part for computation, then formats the results, uploads them to S3 and sends a notification to the API that they are ready.
+The Python part of the worker is a wrapper around the R part: it receives tasks from the API, parses them, sends them to the R part for computation, then formats the results, uploads them to S3 and sends a notification to the API via Redis that they are ready.
```
Comment (aerlaut) [inline README.md]:
```suggestion
The Python part of the worker is a wrapper around the R part: it receives tasks from the API, parses them, sends them to the R part for computation, then formats the results, uploads them to S3 and sends a notification to the API via Redis-backed socket connection that they are ready.
```

**Review by aerlaut — APPROVED** [review]
LGTM. You can merge after resolving the small comments.

**Pull request merged**